[2024-03-27T19:30:20.972+0800] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-27T19:30:20.990+0800] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: update_daily_table_test45.delete_daily_bigquery_external_table_task scheduled__2024-03-27T10:00:00+00:00 [queued]>
[2024-03-27T19:30:21.005+0800] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: update_daily_table_test45.delete_daily_bigquery_external_table_task scheduled__2024-03-27T10:00:00+00:00 [queued]>
[2024-03-27T19:30:21.005+0800] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-03-27T19:30:21.032+0800] {taskinstance.py:1327} INFO - Executing <Task(BigQueryDeleteTableOperator): delete_daily_bigquery_external_table_task> on 2024-03-27 10:00:00+00:00
[2024-03-27T19:30:21.044+0800] {standard_task_runner.py:57} INFO - Started process 3145 to run task
[2024-03-27T19:30:21.056+0800] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'update_daily_table_test45', 'delete_daily_bigquery_external_table_task', 'scheduled__2024-03-27T10:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/flights_ingestion_dag.py', '--cfg-path', '/tmp/tmpn0vq_25p']
[2024-03-27T19:30:21.062+0800] {standard_task_runner.py:85} INFO - Job 10: Subtask delete_daily_bigquery_external_table_task
[2024-03-27T19:30:21.096+0800] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-27T19:30:21.178+0800] {task_command.py:410} INFO - Running <TaskInstance: update_daily_table_test45.delete_daily_bigquery_external_table_task scheduled__2024-03-27T10:00:00+00:00 [running]> on host 2ed6da3b8fe0
[2024-03-27T19:30:21.201+0800] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-27T19:30:21.338+0800] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='update_daily_table_test45' AIRFLOW_CTX_TASK_ID='delete_daily_bigquery_external_table_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-27T10:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-27T10:00:00+00:00'
[2024-03-27T19:30:21.345+0800] {bigquery.py:2348} INFO - Deleting: flight-analytics-418504.flights_stg.external_table_daily_flights
[2024-03-27T19:30:21.348+0800] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-03-27T19:30:21.732+0800] {bigquery.py:1231} INFO - Deleted table flight-analytics-418504.flights_stg.external_table_daily_flights
[2024-03-27T19:30:21.748+0800] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=update_daily_table_test45, task_id=delete_daily_bigquery_external_table_task, execution_date=20240327T100000, start_date=20240327T113020, end_date=20240327T113021
[2024-03-27T19:30:21.825+0800] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-03-27T19:30:21.874+0800] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
